app/src/components/widgets/charts/investigatedTrendChart/legend/legend.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import classNames from 'classnames/bind';
+import PropTypes from 'prop-types';
+import { injectIntl, intlShape } from 'react-intl';
+import * as COLORS from 'common/constants/colors';
+import { MESSAGES } from '../common/constants';
+import styles from './legend.scss';
+
+const cx = classNames.bind(styles);
+
+@injectIntl
+export class Legend extends Component {
```

Comment:
Can `components/widgets/charts/common/legend/legend.jsx` be used instead of this new component?

Reply:
removed